superbench/runner/runner.py

```diff
@@ -405,7 +408,9 @@ def _run_proc(self, benchmark_name, mode, vars):
         if mode.name == 'mpi' and mode.node_num != 1:
             ansible_runner_config = self._ansible_client.update_mpi_config(ansible_runner_config)
 
-        ansible_runner_config['timeout'] = timeout
+        if isinstance(timeout, int):
+            # we do not expect timeout in ansible unless subprocess hangs
+            ansible_runner_config['timeout'] = timeout + 300
```

Comment:
Here the timeout is for async call?

Reply:
nope, previously "timeout x command" and ansible will both kill the processes at the nearly same time, ansible timeout will not collect error/timeout logs for remote processes

now "timeout x command" will kill the processes and log related info, ansible timeout won't be triggered by default unless bash timeout hang for 300 seconds